### DISCUSSION TITLE
[Parquet] is parquet v2 format supported?

### DISCUSSION THREAD

**Discussion by keltia** [duckdb/duckdb#10079] [Q&A]
Hello, reading the code in `parquet_writer.cpp` parquet v2 format doesn't seem to be supported, is it right?

I'm using the `arrow2` crate in Rust to generate my parquet files and they are read without issue by DuckDB so I wonder why only v1 files seem to be written.

**Comment by l1t1** [comment]
https://github.com/datafuselabs/databend/issues/14135
